content/blog/2018/01/2018-01-15-overhaul-of-manage-jenkins-page.adoc

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: "Overhaul of Manage Jenkins page"
+tags:
+- jenkins
+- ui
+- restyling
```

Comment:
I would rather replace it with "upgrade" which we use in other similar cases

Reply:
I've just added `upgrade` as well because `restyling` (IMHO) is a right term to define what we have done here. And we could use in upcoming UI improvements.